wrapanapi/systems/lenovo.py

```diff
@@ -90,6 +90,15 @@ def _service_put(self, path, request):
         except Timeout:
             return None
 
+    def _service_post(self, path, request):
+        """An instance of the service"""
+        try:
+            response = requests.post(self.url + path, data=json.dumps(request), auth=self.auth,
```

Comment:
Please refrain from using string concatenation and form this endpoint with `'{}/{}'.format(self.url, path)`